src/RequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -591,7 +591,6 @@ IN_PROCESS_APPLICATION::RunDotnetApplication(DWORD argc, CONST PCWSTR* argv, hos
     }
     __except (FilterException(GetExceptionCode(), GetExceptionInformation()))
     {
```

Comment:
you may want to add log here. As this failure is critical

Reply:
+1

Reply:
We already log this failure outside of this function https://github.com/aspnet/IISIntegration/blob/dev/src/RequestHandler/inprocess/inprocessapplication.cpp#L556 and when the thread returns https://github.com/aspnet/IISIntegration/blob/dev/src/RequestHandler/inprocess/inprocessapplication.cpp#L472

Reply:
It also needs to be outside of that function because __try, __except can't have any objects that require object unwinding, like STRU.

Reply:
OK cool 👍